client/src/Forms/DefineForm.js

```diff
@@ -1,13 +1,155 @@
 import React from 'react';
+import Form from './FormComponent';
+import {theme, ThemeProvider} from "../Libraries/ReactToolboxLibrary";
 
-import { add_definition_url } from "../urls";
+import '../stylesheets/defineForm.css';
 
-const DefineForm = () => {
+const snackbarMessage = "One or more required fields are blank";
+const submissionFailureMessage = "Oh no! Something went wrong, please try again.";
+
+const submissionSuccessMessage = (termName) => {
+  return `Your proposed definition for ${termName} was submitted successfully`
+};
+
+export default class DefineForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      submitterName: "",
+      termName: "",
+      termDefinition: "",
+      termMeaningToSubmitter: "",
+      submitterIdentities: "",
+      stateBar: false,
+      snackbarMessage: ''
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(label, isRequired, showErrorInput, value) {
+    this.setState({[label]: value});
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    if(this.formChild.hasErrors()) {
```

Comment:
Suboptimal to get this from a child element but I want to save refactoring this away for a separate PR.